### PR TITLE
Removes hard coded secrets

### PR DESCRIPTION
**Pull request by omawhite**
### What
Removes hard coded secrets and uses the dotenv plugin for local development.

### Why
Security mainly, this is a bit better than just having these values in our index.html

### Ensure the following interactions work as expected. Please test using a mobile form factor.
- [ ] Tapping on a marker on the map displays information about the marker in a popup.
- [ ] Tapping the "Show list of locations" button replaces the map view with a list view.
- [ ] Tapping an item in the list replaces the list view with a map view, and navigates the map to the tapped item on the map.
- [ ] Tapping one of the ✅'s in the legend filters items on the map and in the list.

### Check the app in the following web browsers:
- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ To get started, you have to install the dependencies (mainly snowpack)
 npm install
 ```
 
+### Environment Labels
+In order to run locally you will need to retrieve the `.env` file from here.........location pending
```
Comment (alexives) [inline README.md:44]:
```suggestion
In order to run locally you will need to retrieve the `.env` file from someone in the Open Twin Cities slack `#tc-aid-dev` channel.
```

**Comment by experimatt** [comment]
@omawhite I forgot to click "Save" when I updated the variable names in Netlify late last night, but it should be fixed now. I just kicked off a re-deploy of the netlify preview which will hopefully fix it so we can confirm that everything's working as expected.

File: src/config.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
+const { SNOWPACK_PUBLIC_MAPBOXGL_ACCESS_TOKEN, SNOWPACK_PUBLIC_DATA_URL, SNOWPACK_PUBLIC_TRANSLATION_URL} = import.meta.env
+
+
 export default {
-  dataUrl: 'https://spreadsheets.google.com/feeds/list/1CyPozeKhmOuIaVnKDQAUKKsvA9R4F20hEQ3MSGWczP8/1/public/full?alt=json',
-  accessToken: 'pk.eyJ1IjoiamFjb2JyZGFsdG9uIiwiYSI6ImNrYXc2anFjbjIxNGwyeG14dWk0MDVycmIifQ.yILWplracVMB4mv1dWwTtg',
-  translationUrl: 'https://spreadsheets.google.com/feeds/list/1m5QPNP6O8nsQsqJcQbwib_obpXRjmbYYGtwKzzg1l38/1/public/full?alt=json'
+  dataUrl: SNOWPACK_PUBLIC_DATA_URL,
```
Comment (alexives) [inline src/config.js:5]: We should try to add some kind of warning if these don't exist. Might want something like:
```diff
diff --git a/src/config.js b/src/config.js
index 69b0bda..62de88c 100644
--- a/src/config.js
+++ b/src/config.js
@@ -1,8 +1,14 @@
 const { SNOWPACK_PUBLIC_MAPBOXGL_ACCESS_TOKEN, SNOWPACK_PUBLIC_DATA_URL, SNOWPACK_PUBLIC_TRANSLATION_URL} = import.meta.env
 
+function ensure(variable_name, variable) {
+  if(import.meta.env.MODE === 'development') {
+    console.error(`${variable_name} must be defined in your environment. See https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations#environment-labels for details`)
+  }
+  return variable
+}
 
 export default {
-  dataUrl: SNOWPACK_PUBLIC_DATA_URL,
-  accessToken: SNOWPACK_PUBLIC_MAPBOXGL_ACCESS_TOKEN,
-  translationUrl: SNOWPACK_PUBLIC_TRANSLATION_URL
+  dataUrl: ensure("SNOWPACK_PUBLIC_DATA_URL", SNOWPACK_PUBLIC_DATA_URL),
+  accessToken: ensure("SNOWPACK_PUBLIC_MAPBOXGL_ACCESS_TOKEN", SNOWPACK_PUBLIC_MAPBOXGL_ACCESS_TOKEN),
+  translationUrl: ensure("SNOWPACK_PUBLIC_TRANSLATION_URL", SNOWPACK_PUBLIC_TRANSLATION_URL)
 }
```

File: README.md (hunk 1 above)
Comment (experimatt) [inline README.md:44]: This is a very good point. If nothing else, I'd like to see a `.env.example` file with the keys that we're expecting, even with dummy values. We'll have to figure out how to share these "secrets" with the whole team

Comment (omawhite) [reply README.md:44]: good suggestion for now

File: src/config.js (hunk 2 above)
Comment (omawhite) [reply src/config.js:5]: nice suggestion!

**Comment by omawhite** [comment]
@experimatt @alexives made some changes.

File: README.md [hunk 3]
```diff
@@ -40,6 +40,9 @@ To get started, you have to install the dependencies (mainly snowpack)
 npm install
 ```
 
+### Environment Labels
```
Comment (alexives) [inline README.md:43]: There's a slight change to the setup instructions in master, and I wonder if this actually belongs in there. If you merge master in, you could add a step, like:
```diff
diff --git a/README.md b/README.md
index cfee13c..1b80c18 100644
--- a/README.md
+++ b/README.md
@@ -35,14 +35,12 @@ This started as a very lightweight, single page html file, and we've tried very
 ### Setup
 
 1. Install node at the version specified in the [`.node_version`](.node_version) file. If you use a version manager like nodenv or nvm, this should be detected automatically.
-2. Install dependencies with npm
+2. Create a `.env` file with secrets needed to run the app. You can get the secrets you need from someone in the Open Twin Cities slack `#tc-aid-dev` channel.
+3. Install dependencies with npm
     ```
     npm install
     ```
-3. Start building!
-
-### Environment Labels
-In order to run locally you will need to retrieve the `.env` file from someone in the Open Twin Cities slack `#tc-aid-dev` channel.
+4. Start building!
 
 ### Run locally
```

Comment (omawhite) [reply README.md:43]: sounds fine to me is there a way we can just commit this suggestion?

Comment (alexives) [reply README.md:43]: If you merge master into the branch I can convert it from a diff to a suggestion

Comment (omawhite) [reply README.md:44]: done

Comment (experimatt) [reply README.md:44]: I think we can tidy up the readme instructions after merging this in. No point in going back and forth with more review

**Comment by experimatt** [comment]
@omawhite this looks good except the new `ensure()` function may have broken something. Testing it out with the Netlify deploy preview shows something is broken: https://deploy-preview-93--wonderful-montalcini-96d80d.netlify.app/

**Comment by experimatt** [comment]
This PR is currently in limbo at the moment. Omar's offline for the rest of the day, and I will be shortly as well. Anyone should feel welcome to pick this up and run with it to take it across the finish line.

**Comment by experimatt** [comment]
Related issue for updating the readme: https://github.com/Twin-Cities-Mutual-Aid/twin-cities-aid-distribution-locations/issues/102

**Pull request merged**